src/assets/css/scss/layout/_hero.scss

```diff
@@ -181,7 +181,7 @@
     }
 
     .hero-illustration {
-        right: 0;
+        right: -160px;
```

Comment:
It was actually a really easy fix since everything had relative styling so all that was needed was to tweak the right offset. This will only affect large displays, which means it still scales well and does not affect mobile.

Reply:
Any chance we can get this fix working on mobile too? We can't confirm if our users will be on desktop or mobile so we have to make sure it looks pretty good on both